.github/workflows/ci-actions.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java-version: [8, 11, 12]
+        java-version: [8, 11, 13]
```

Comment:
Hum. Apparently, we get a 404 for that one whereas it's supposed to be supported.

Reply:
Got back to 12 as I got a 404 with 13.